components/accept-financial-contributions/AcceptContributionsMyselfOrOrg.js

```diff
@@ -81,10 +81,10 @@ class AcceptContributionsMyselfOrOrg extends React.Component {
   // GraphQL functions
   addHost = async (collective, host) => {
     const collectiveInput = {
-      legacyId: collective.id,
```

Comment:
slug is compatible with GraphQL API v1 and v2, so we don't need to think about it